common/src/main/java/org/wso2/testgrid/common/config/InfrastructureConfig.java

```diff
@@ -124,6 +124,7 @@ public void setIncludes(List<String> includes) {
     public enum IACProvider {
         CLOUDFORMATION("CloudFormation"),
         TERRAFORM("Terraform"),
+        KUBERNETES("Kubernetes"),
```

Comment:
Why the DeployerType is in uppercase, but here we have lowercase 'Kubernetes' key? It seems we have the same issue for other keys as well, but let's follow the right approach from now on.

https://github.com/wso2/testgrid/pull/1210/files#diff-041aaecd98ee0a1c15effef3439ab729R621